routes/api-reference-route.tsx

```diff
@@ -62,76 +59,28 @@ export function apiReferenceRoute({
 
         const pages = docs["."];
 
-        const page = docs["."].find((node) => node.name === symbol);
+        const page = pages.find((node) => node.name === symbol);
 
         if (!page) throw new Error(`Could not find a doc page for ${symbol}`);
```

Comment:
A nice thing about Effection, no need to use Errors as continuations since we can generalize over _all_ categories of continuation. In this case: Http Not Found: https://github.com/thefrontside/revolution/blob/ee964c4f5e5d425c085c06d7ac01755d32e3bf2e/lib/middleware/http-responses.ts#L14-L23

 ```ts
import { respondNotFound } from "revolution";

// .....
if (!page) {
  yield* respondNotFound(); // control passes immediately to the response middleware
}

Reply:
Can you please tell me if this looks right before I apply it to all of the routes? https://github.com/thefrontside/effection-www/pull/4/commits/d4ff68ec0f1b0d685d0e60decaba4c1645522679